sources/curriculum/0_before_you_start/workflows-and-pipelines-detailed/README.md

```diff
@@ -0,0 +1,194 @@
+# Pipelines and Project Workflow
+
+## Motivation
+
+Most data-science projects have the same set of tasks:
+
+1. **ETL**: extracting data from its source, transforming it, then loading it into a database.
+2. **Pre-process data**: This might include imputing missing values and choosing the training and testing sets.
```

Comment:
What about feature generation?

Reply:
Agreed. I just copied this from the old version. Following your comment, I have not combined the two versions (old and new) and added feature creation to the list.